Docs/Request_Overrides.md

```diff
@@ -255,4 +255,46 @@ Example:
     {status: "fail"}
   ]
 }
+```
+
+## getAppState:
+
+### Request override
+- The `getAppState` request override Sends message to the platform to call the WebSocket handler and to retrieve the status of all apps.. This function has to be added in config module `requestModules/fcs.js` file.
+
```

Comment:
you should not mention about sending message to call websocket handler. you are giving bolt information here which is comcast proprietary